bin/build

```diff
@@ -1,3 +1,18 @@
 #!/usr/bin/env bash
 
-docker build -t arnold:$(tr -d '\n' < VERSION) .
+IMAGE="arnold:$(tr -d '\n' < VERSION)"
+
+# Dockerlint before build
+docker run --rm -i hadolint/hadolint < Dockerfile
+docker build -t "${IMAGE}" .
+
+# Other lint after build
+docker run --rm -it \
+  -e OC_LOGIN=false \
+  "${IMAGE}" \
+  bin/lint ansible
+
+  docker run --rm -it \
+    -e OC_LOGIN=false \
+    "${IMAGE}" \
+    bin/lint bash
```

Comment:
So isn't the lint going to be done twice in the CI if you also do it when we call build?
My feeling is that build should do build and lint should do lint.

Reply:
`bin/build` is a sugar script only for dev is not call bye the CI.